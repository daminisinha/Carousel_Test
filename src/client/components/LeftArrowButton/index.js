import React, { Component } from 'react';
import './index.scss'
export default class LeftArrow extends Component {
  
  render() {
    return (
     
      <img onClick={this.props.prevImage} alt="Left Arrow" width="20" height="20" className="left-arrow" src="./images/arrow.svg" alt="Left Arrow"/>
    
    );
  }
}

