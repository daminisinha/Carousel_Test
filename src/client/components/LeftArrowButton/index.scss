.left-arrow {
    transform: rotate(180deg);
    max-width: 100%;
    max-height: 100%;
    height: auto;
    cursor: pointer;
    position: absolute;
    border-top-left-radius: 50px;
    border-bottom-left-radius: 50px;
    padding:10px;
    background-color:whitesmoke;
    filter: grayscale(100%);
    top: 30%;
    z-index: 999;
    float: left;
    opacity: 0.5;
}