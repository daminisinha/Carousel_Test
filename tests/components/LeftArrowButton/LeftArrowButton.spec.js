import React from 'react';
import LeftArrowButton from '../../../src/client/components/LeftArrowButton'
import {shallow} from 'enzyme';

test('Left arrow button render renders',()=>{
    const component = shallow(<LeftArrowButton/>); 
    expect(component).toMatchSnapshot();
})
