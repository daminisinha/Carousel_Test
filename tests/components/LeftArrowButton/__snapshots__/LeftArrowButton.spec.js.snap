// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Left arrow button render renders 1`] = `
<img
  alt="Left Arrow"
  className="carousel__left__arrow"
  height="20"
  src="./images/arrow.svg"
  width="20"
/>
`;
